tests/ha/test_multi_server_pod_failure.py

```diff
@@ -629,3 +629,584 @@ def test_writes_till_kserver_pods_fail(self):
+                                                          f"object {key} of bucket {val}.")
+        LOGGER.info("Step 8: Successfully download the uploaded objects & verify etags")
+
+        bucketnew = f"ha-bkt-new-{int((perf_counter_ns()))}"
+        objectnew = f"ha-obj-new-{int((perf_counter_ns()))}"
+        bkt_obj_dict.clear()
+        bkt_obj_dict[bucketnew] = objectnew
+        LOGGER.info("Step 9: Perform copy of %s from already created/uploaded %s to %s and verify "
+                    "copy object etags", self.object_name, self.bucket_name, bucketnew)
+        resp = self.ha_obj.create_bucket_copy_obj(event, s3_test_obj=s3_test_obj,
+                                                  bucket_name=self.bucket_name,
+                                                  object_name=self.object_name,
+                                                  bkt_obj_dict=bkt_obj_dict,
+                                                  put_etag=put_etag,
+                                                  bkt_op=False)
+        assert_utils.assert_true(resp[0], f"Already created/uploaded copy "
+                                          f"object failed buckets are: {resp[1]}")
+        LOGGER.info("Step 8: Performed copy of %s from already created/uploaded %s to %s and "
+                    "verified copy object etags", self.object_name, self.bucket_name, bucketnew)
+
+        LOGGER.info("Step 9: Download the uploaded %s on %s & "
+                    "verify etags.", objectnew, bucketnew)
+        resp = s3_test_obj.get_object(bucket=bucketnew, key=objectnew)
+        LOGGER.info("Get object response: %s", resp)
+        get_etag = resp[1]["ETag"]
+        assert_utils.assert_equal(put_etag, get_etag, "Failed to match GET-PUT ETAG "
+                                                      f"for object {objectnew} of bucket "
+                                                      f"{bucketnew}.")
+        LOGGER.info("Step 9: Downloaded the uploaded %s on %s & verified etags.",
+                    objectnew, bucketnew)
+
+        LOGGER.info("COMPLETED: Verify copy object during server pods failure "
+                    "till K pods down - unsafe shutdown")
+
+    # pylint: disable=C0321
+    # pylint: disable=too-many-locals
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-40576")
+    @CTFailOn(error_handler)
+    def test_deletes_during_kserver_pods_down(self):
+        """
+        This test tests continuous DELETEs while pods are failing till K server
+        pods down - unsafe shutdown
+        """
+        LOGGER.info("STARTED: Test to verify continuous DELETEs while pods are failing till K "
+                    "server pods down - unsafe shutdown.")
+        event = threading.Event()  # Event to be used to send intimation of server pod deletion
+        wr_output = Queue()
+        del_output = Queue()
+        wr_bucket = self.kvalue * 5 + HA_CFG["s3_bucket_data"]["no_buckets_for_deg_deletes"]
```

Comment:
150 would be fine. We only need this logic in till K loop, scenario

Reply:
for now keeping as it is.